lib/Conversion/StandardToLLVM/ConvertStandardToLLVM.cpp

```diff
@@ -238,13 +238,26 @@ class LLVMLegalizationPattern : public LLVMOpLowering {
   }
 
   // Get the array attribute named "position" containing the given list of
-  // integers as integer attribute elements.
+  // integers as integer attribute elements of index type.
+  static ArrayAttr getIndexArrayAttr(ConversionPatternRewriter &builder,
+                                     ArrayRef<int64_t> values) {
+    SmallVector<Attribute, 4> attrs;
+    attrs.reserve(values.size());
+    for (int64_t pos : values)
+      attrs.push_back(builder.getIntegerAttr(builder.getIndexType(), pos));
+    return builder.getArrayAttr(attrs);
+  }
+
+  // Get the array attribute named "position" containing the given list of
+  // integers as integer attribute elements of width "bitwidth".
   static ArrayAttr getIntegerArrayAttr(ConversionPatternRewriter &builder,
+                                       unsigned bitWidth,
```

Comment:
Is this function necessary anymore? It should be covered by builder.get* methods for most cases.

Reply:
Not needed anymore.